src/components/icons/BrowserIcon.jsx

```diff
@@ -0,0 +1,38 @@
+import createIcon from './createIcon'
+
+export default createIcon(({ size, color, ...props }) => (
```

Comment:
Grommet actually has a nice icon wrapper we can use, and I also paired it with airbnb's svg loader in our old, poorly-designed component library that you might want to imitate: https://github.com/pluralsh/forge-core/blob/master/src/components/icons.js

Leveraging it will give us small/medium/large size theming plus color theming for free basically, plus a nice experience for adding new icons

Reply:
Definitely look it up